src/render.c

```diff
@@ -91,7 +91,7 @@ void render_init (Render* render, Args* args)
 	float scale = MIN(
 			display->screen->w/(float)BASE_W,
 			display->screen->h/(float)BASE_H
-		) * args->antialiasing ? 2 : 1;
+		) && args->antialiasing ? 2 : 1;
```

Comment:
This seem wrong. I can't see how it could work. What was the compiler error you were trying to fix here?

Reply:
I see. But you shouldn't follow compiler suggestions blindly. The render.c:94 "error" should be solved with a parenthesis instead, like `* (args->antialiasing ? 2 : 1)`.  Using `&&` makes no sense in this context.

Reply:
I fixed the errors myself. Thanks for bringing them to my attention. I haven't compiled cave9 for years, and GCC is more pedantic nowadays.